community/binutils-avr/APKBUILD

```diff
@@ -2,7 +2,7 @@
 # Maintainer: Stefan Wagner <stw@bit-strickerei.de>
 pkgname=binutils-avr
 pkgver=2.32
-pkgrel=0
+pkgrel=1
```

Comment:
Is there a reason for the pkgrel bump here? As far as I know the secfixes don't impact the package, and removing the `-j1` etiher.

Reply:
No, it doesn't really matter here.